### PR TITLE
Split out index operations into own script

### PR DESCRIPTION
**Pull request by wpears**
I often have to reindex locally (when the python container crashes due to some hilarious python error I've made, or just when  I turn docker off and on).

This makes that process easier, allowing the index operations to be called independently of `refresh-data.sh`.

### CODE REVIEW

File: refresh-data.sh [hunk 1]
```diff
@@ -69,9 +69,7 @@ refresh_data() {
 }
 
 update_index() {
-    echo 'Updating search indexes'
-    ./cfgov/manage.py opensearch index --force rebuild
-    ./cfgov/manage.py opensearch document --force --refresh --parallel index
+  source ./index.sh
```
Comment (chosak) [inline refresh-data.sh:72]: Since index.sh is chmoded+x and has a `#!/bin/bash`, does this need to be sourced, or should it just be called?

**Review by chosak — COMMENTED** [review]
Could you update the documentation, e.g. [here](https://cfpb.github.io/consumerfinance.gov/page-search/#building-the-index)?

File: refresh-data.sh (hunk 1 above)
Comment (wpears) [reply refresh-data.sh:72]: I `source`d it so that it would run in the same process, which maybe is unnecessary, but seemed reasonable

**Comment by wpears** [comment]
> Could you update the documentation, e.g. [here](https://cfpb.github.io/consumerfinance.gov/page-search/#building-the-index)?

Done in 1e374b5c1e

**Review by chosak — APPROVED** [review]
🪓

**Pull request merged**